pkg/runtime/wfengine/activity.go

```diff
@@ -78,19 +78,30 @@ func (a *activityActor) InvokeMethod(ctx context.Context, actorID string, method
 	}
 
 	// Try to load activity state. If we find any, that means the activity invocation is a duplicate.
-	if state, err := a.loadActivityState(ctx, actorID, ar.Generation); err != nil {
+	var state activityState
```

Comment:
Uhm something feels odd here. `state` is not read from `a.loadActivityState` anymore, so it will always be empty? and `if state.Generation> 0` will always be false? I'm not sure what this should have been.